lib/index.test.js

```diff
@@ -129,11 +130,29 @@ test('should render invisible recaptcha', t => {
       callback: onVerify,
       'expired-callback': onExpire,
       'error-callback': onError,
-      isolated: true
+      isolated: true,
+      hl: null
     })
   );
 });
 
+test('should render recaptcha with hl', t => {
+  t.plan(1);
+
+  mount(
+    <Reaptcha
+      sitekey="my-key"
+      hl="fr"
+    />
+  );
+
+  t.true(
```

Comment:
does this test case make sense?

Reply:
Yes, this test is cool now :) Just change the the above comment please :)